### ISSUE TITLE
Mandate using RelWithDebugInfo for development builds on our website

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#3035]
I think it would be good if we had a policy that all builds which go on our website which are not stable releases should be compiled using RelWithDebugInfo. That would end problems with users not being able to find debug builds to send stack backtraces, and it would strengthen the distinction between our stable releases and the development builds. Last I tried, a few years ago, RelWithDebugInfo ran about as fast as Release. The question is how useful are its debug symbols?

**Comment by adamreichold** [comment]
I think the debug symbols are very useful, but most Unix package formats like RPM or DEB should already do that by default, so this should apply mostly to Windows users? And they would still need to install the tool chain up to GDB, so that getting a debug build is probably the smaller part of creating a back trace for the average Windows user?

Another issue might be that stack traces from highly optimized code are sometimes hard to parse due to heavy inlining. So overall I'd say a policy that each build we publish should contain a debug and a release binary built from the same sources might even be more useful?

**Comment by sguyader** [comment]
From now on, I'll try to always provide debug builds along my release builds. It doesn't cost me much more than a few minutesof building and uploading.

**Comment by Beep6581** [comment]
Then let's close this.
